content/e20co/drone-marathon/index.md

```diff
@@ -0,0 +1,103 @@
+---
+author: "dellintosh"
+date: 2017-01-06T18:00:00-06:00
+image: "e20co/drone-marathon"
+repo: "e20co/drone-marathon"
```

Comment:
@appleboy We are still using 0.4 in several of our company projects; so I'll need to keep the `master` branch (0.4) as it is for now... once we update our codebase(s) to use the new version I'll merge that into master.  ;)